include/transport/transport.h

```diff
@@ -9,6 +9,8 @@
 #include <sys/socket.h>
```

Comment:
Without `<array>` included here we hit in `libquicr`:

```
error: constexpr variable cannot have non-literal type 'const std::array<uint8_t, sizeof(uint64_t)>' (aka 'const array<unsigned char, sizeof(uint64_t)>')
```

Reply:
Thanks. Fixed with latest commit.